src/components/CardView.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from "react";
+import {Header} from "semantic-ui-react";
+import {Helmet} from "react-helmet";
+import {Map, TileLayer, Marker, Popup, Polyline} from "react-leaflet";
+import {connect} from "react-redux";
+
+const zoomLevel = 13;
+
+class Card extends Component {
+    render() {
+        const center = [51.505, -0.09]
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "center" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'center' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)